package.json

```diff
@@ -36,6 +38,7 @@
     "express": "^4.17.1",
     "husky": "^3.0.3",
     "node-fetch": "^2.6.0",
+    "pm2": "^3.5.1",
```

Comment:
should not be in node modules. `pm2` should be install globally during setup step

Reply:
Why we can't install it in `node_modules` ? 
To install `pm2` globally we have to be log as root or ask the final user to run sudo... (not good)
Or we have to tell them within Readme instruction.